### ISSUE TITLE
Some reproduction details of MFNet

### ISSUE THREAD

**Issue by tianqiong123** [taotaowang97479/MFNet-SpeechEnhancement#2]
Hi! I'm very interested in MFNet and I'm going to reproduce it recently. But I don't know the  implementation details
about the gate layer and the channel attention layer of MFNet from the paper "a Mask Free Neural Network for Monaural Speech Enhancement".
I would greatly appreciate it if you would like to tell me about the process and hyperparameters of gate layer and the channel attention layer in MFNet.

**Comment by taotaowang97479** [comment]
我使用了作者评论区修改后的网络代码，你可以参考：https://github.com/ioyy900205/MFNet/issues/1

**Comment by tianqiong123** [comment]
好的，非常感谢
